python/pyspark/mllib/tests.py

```diff
@@ -789,6 +790,29 @@ def test_model_transform(self):
         self.assertEqual(model.transform([1.0, 2.0, 3.0]), DenseVector([1.0, 2.0, 3.0]))
 
 
+class MLUtilsTests(MLlibTestCase):
+    def test_append_bias(self):
+        data = [1.0, 2.0, 3.0]
+        ret = MLUtils.appendBias(data)
+        self.assertEqual(ret[3], 1.0)
+
+    def test_load_vectors(self):
+        import shutil
+        data = [
+            [1.0, 2.0, 3.0],
+            [1.0, 2.0, 3.0]
+        ]
+        try:
+            self.sc.parallelize(data).saveAsTextFile("test_load_vectors")
+            ret_rdd = MLUtils.loadVectors(self.sc, "test_load_vectors")
+            ret = ret_rdd.collect()
```

Comment:
Order of collect() is not guaranteed, so please sort "ret" and "data" and then compare to make the test robust.